check-your-phone.md

```diff
@@ -0,0 +1,2 @@
+You look up and suddenly, there is a woman in 
```

Comment:
@cbuckey-uda thank you so much. i deleted the file connected to my first two erroneous commits, and now my pull request indicates "Files changed (1)." my latest commit "Add sentence to check-your-phone" (52579b9) added a sentence to the correct place. i hope i did the right thing.